app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,11 @@
+%h1 Log In
+= form_for :session, :url => "/login" do |f|
+  = f.label :email
+  %br
+  = f.text_field :email
```

Comment:
agreed, will update the user model to validate presence of email, first_name, and last_name